### ISSUE TITLE
SQL Statements should be parametised

### ISSUE THREAD

**Issue by nohumanman** [nohumanman/descenders-modkit#28]

**Comment by nohumanman** [comment]
Additionally, sql statements should not have a function each - epsecially with aiosql .

aiosql should load qeuries like so:

```
queries = aiosql.from_path("queries.sql", "sqlite3")

with sqlite3.connect("modkit.db") as conn:
    user = queries.get_personal_fastest_split_times(conn, trail="willvaughn", steam_id=123098120938019283)
```

**Comment by nohumanman** [comment]
Fixed in [410448c](https://github.com/nohumanman/descenders-modkit/commit/410448ceac951c2ce86f02a6bbc1573ad92e2777)
